.ocamlformat

```diff
@@ -0,0 +1,9 @@
+profile=ocamlformat
+version=0.14.1
```

Comment:
I think it's better if we remove the ocamlformat version out, we're updating it very soon.